src/main/java/app/App.java

```
@@ -1,14 +1,26 @@
 package app;
 
+import app.telegram.TelegramBot;
 import lombok.RequiredArgsConstructor;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.telegram.telegrambots.meta.TelegramBotsApi;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;
 
 @SpringBootApplication
 @RequiredArgsConstructor
 public class App {
     public static void main(String[] args) {
-        SpringApplication.run(App.class, args);
+        ApplicationContext context = SpringApplication.run(App.class, args);
+        TelegramBot telegramBot = context.getBean(TelegramBot.class);
+
+        try {
+            TelegramBotsApi botsApi = new TelegramBotsApi(DefaultBotSession.class);
+            botsApi.registerBot(telegramBot);
+        } catch (TelegramApiException e) {
+            e.printStackTrace();
```

maybe it should be thrown with custom exception instead of printStackTrace()